docs/config.md

```diff
@@ -370,8 +370,7 @@ system, such as */tmp* in Linux systems.
 
 ### Configuring TaskManager processing slots
 
-A processing slot allows Flink to execute a distributed DataSet transformation, such as a
-data source or a map-transformation.
+A processing slot allows Flink to execute an instance of a job.
```

Comment:
I think the term *instance of a job* might be misleading. The document uses *operator instance* or *(user) function instance* but IMO it is not clear what an *instance of a job* is.

I agree that the previous description wasn't good either, but we might try to come up with something better.
What about "Flink executes a program in parallel by splitting it into subtasks and scheduling these subtasks to processing slots."?